pages/docs/custom-dashboards.mdx

```diff
@@ -0,0 +1,65 @@
+---
+description: Create self-service dashboards to visualize, monitor, and share insights from your LLM application data.
+---
+
+# Custom Dashboards
+
+<AvailabilityBanner
+  availability={{
+    hobby: "public-beta",
+    core: "public-beta",
+    pro: "public-beta",
+    enterprise: "public-beta",
+    selfHosted: "not-available",
+  }}
+/>
+
+Langfuse custom dashboards allow you to create personalized views of your LLM application metrics.
+Build widgets that visualize specific aspects of your data and arrange them into comprehensive dashboards
+to monitor performance, usage patterns, and other important metrics.
+
```

Comment:
add sentence regarding that this is currently in beta + link to main GH discussions thread on this for feedback